### ISSUE TITLE
Prow: stop using volumes and use local disk instead

### ISSUE THREAD

**Issue by tuminoid** [metal3-io/project-infra#922]
With the new private infrastructure servers deployed, Prow should stop using slow volumes and use local disk instead.

**Comment by tuminoid** [comment]
@lentzi90 as discussed offline.

**Comment by lentzi90** [comment]
Let's just confirm that we have capacity first. It would not be nice to reserve local disk for prow if that makes the jenkins tests starved for resources.

**Comment by tuminoid** [comment]
For the reference, can you summarize how the disk space was spent right now? @lentzi90

**Comment by lentzi90** [comment]
Sure! Prow is currently using [100 GiB](https://github.com/metal3-io/project-infra/blob/ae5d38f7cdd98bb460c73efdd166c7ce00ac43f4/prow/capo-cluster/openstackmachinetemplates.yaml#L70) volumes. The cluster consists of 1 control plane node and minimum 3 worker nodes at the moment. The workers can scale up to a total of 6. The bastion host does not use any volume.

**In total, that can be maximum 700 GiB. Most of the time we are at 400 GiB.**

(Outside of this is 1 volume for ghproxy, also 100 GiB. This is attached to a pod so using local disk for it may not be a good idea. It would lose all state every image bump or eviction.)

**Comment by tuminoid** [comment]
Thank you. I think we are safe to do the change next week. Leave ghproxy with a volume.
